05_syntax_lab/06.py

```diff
@@ -1,9 +1,8 @@
-"""
-Write a program that randomizes 2 numbers
-and calculates their least common multiplier,
-that is the smallest number that is divisable
-by both.
-For example if the numbers were 4 and 6,
-program should print 12
-"""
-
+#Week01_Exercise01_Assignment06
+from random import randint
+x,y,z = randint(1,10),randint(1,10),1
+for prime in [2,3,5,7]:
+    pwr = 0
+    while (x%(prime**(pwr+1))) * (y%(prime**(pwr+1))) == 0 : pwr += 1
+    z *= prime**pwr
+print "The smallest number devided by [%d] and [%d] is [%d]" % (x,y,z)

```

Comment:
Python also has a gcd function (inside module fractions). Can you simplify the code using that?